Resources/Prototypes/Entities/Objects/Consumable/Food/meat.yml

```diff
@@ -0,0 +1,915 @@
+  - type: SliceableFood
+    count: 3
+    slice: FoodMeatLizardCutlet
+
+- type: entity
+  name: raw plant meat
+  parent: FoodMeatBase
+  id: FoodMeatPlant
+  description: All the joys of healthy eating with all the fun of cannibalism.
+  components:
+  - type: Sprite
+    state: plant
+
+- type: entity
+  name: rotten meat
+  parent: FoodMeatBase
+  id: FoodMeatRotten
+  description: Halfway to becoming fertilizer for your garden.
+  components:
+  - type: Tag
+    tags:
+      - Raw
+  - type: Sprite
+    state: rotten
+  - type: SolutionContainer
+    contents:
+      reagents:
+      - ReagentId: chem.Toxin
+        Quantity: 20
+
+- type: entity
+  name: raw spider meat
+  parent: FoodMeatBase
+  id: FoodMeatSpider
+  description: A slab of spider meat. That is so Kafkaesque.
+  components:
+  - type: Tag
+    tags:
+      - Raw
+  - type: Sprite
+    state: spider
+  - type: SolutionContainer
+    contents:
+      reagents:
+      - ReagentId: chem.Toxin
+        Quantity: 20
+  - type: SliceableFood
+    count: 3
+    slice: FoodMeatSpiderCutlet
+
+- type: entity
+  name: raw spider leg
+  parent: FoodMeatBase
+  id: FoodMeatSpiderLeg
+  description: A still twitching leg of a giant spider... you don't really want to eat this, do you?
+  components:
+  - type: Tag
+    tags:
+      - Raw
+  - type: Sprite
+    state: spiderleg
+  - type: SolutionContainer
+    contents:
+      reagents:
+      - ReagentId: chem.Toxin
+        Quantity: 10
+
+- type: entity
+  name: meatwheat clump
+  parent: FoodMeatBase
+  id: FoodMeatWheat
+  description: This doesn't look like meat, but your standards aren't that< high to begin with.
+  components:
+  - type: Tag
+    tags:
+      - Raw
+  - type: Sprite
+    state: clump
+  - type: SolutionContainer
+    contents:
+      reagents:
+      - ReagentId: chem.Toxin
+        Quantity: 1
+
+- type: entity
+  name: raw xeno meat
+  parent: FoodMeatBase
+  id: FoodMeatXeno
+  description: A slab of xeno meat, dripping with acid.
+  components:
+  - type: Tag
+    tags:
+      - Raw
+  - type: Sprite
+    state: xeno
+  - type: SolutionContainer
+    contents:
+      reagents:
+      - ReagentId: chem.SulfuricAcid
+        Quantity: 20
+  - type: SliceableFood
+    count: 3
+    slice: FoodMeatXenoCutlet
+
+- type: entity
+  name: killer tomato meat
+  parent: FoodMeatBase
+  id: FoodMeatTomato
+  description: A slice from a huge tomato.
+  components:
+  - type: Sprite
+    state: tomato
+  - type: SliceableFood
+    count: 3
+    slice: FoodMeatTomatoCutlet
+
+- type: entity
+  name: salami
+  parent: FoodMeatBase
+  id: FoodMeatSalami
+  description: A large tube of salami. Best not to ask what went into it.
+  components:
+  - type: Tag
+    tags:
+      - Raw
+  - type: Sprite
+    state: salami
+  - type: SliceableFood
+    slice: FoodMeatSalamiSlice
+
+- type: entity
+  name: meat clown
+  parent: FoodMeatBase
+  id: FoodMeatClown
+  description: A delicious, round piece of meat clown. How horrifying.
```

Comment:
Should this be "clown meat"? If so, don't forget to change the name too.

Reply:
No, it's a big salami with a clown face on it.